dev-ruby/public_suffix/public_suffix-3.1.1-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="public_suffix.gemspec"
+RUBY_FAKEGEM_RECIPE_DOC="rdoc"
```

Comment:
This is now default in EAPI=7 so it can be removed.

Reply:
Ok upgraded